server/src/main/java/org/opensearch/rest/action/admin/cluster/RestClusterStatsAction.java

```diff
@@ -64,9 +100,95 @@ public String getName() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
-        ClusterStatsRequest clusterStatsRequest = new ClusterStatsRequest().nodesIds(request.paramAsStringArray("nodeId", null));
+        String[] nodeIds = request.paramAsStringArray("nodeId", null);
+        Set<String> metrics = Strings.tokenizeByCommaToSet(request.param("metric", "_all"));
+        Set<String> indexMetrics = Strings.tokenizeByCommaToSet(request.param("index_metric", null));
```

Comment:
Can we make the default for indexMetrics as `_all`?

Reply:
This is null to handle cases when the client request doesn't include indices metrics.